fractal_server/app/schemas/_validators.py

```diff
@@ -10,7 +10,7 @@ def valstr(attribute: str, accept_none: bool = False):
     """
 
     def val(string: str):
-        if string is None:
+        if (string is None) or (string == ""):
```

Comment:
This cannot be like this, since `accept_none` has the side effect of accepting an empty string.

Reply:
I reverted this update, and brought back `test_task_update` to its `main` version. There is not reason why this PR would need to modify the expected behavior of `TaskUpdate`.